text/0000-pack-manifest-list-commands.md

```diff
@@ -0,0 +1,301 @@
+- Image Index: The image index is a higher-level manifest which points to specific image manifests, ideal for one or more platforms. See [spec](https://github.com/opencontainers/image-spec/blob/main/image-index.md) 
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?
+
+The uses of ARM architecture in the cloud and edge computing has been growing rapidly. The CNCF community has been also growing in the last years, and there is a need to support multi-arch for all the projects. The buildpacks community is not an exception, issues like:
+- [It would be nice to support easily creating a manifest list packages and builders](https://github.com/buildpacks/pack/issues/1460)
+- [Provide a way to specify desired platform when creating packages and builders](https://github.com/buildpacks/pack/issues/1459)
+- [Multi arch image build support](https://github.com/buildpacks/pack/issues/1570)
+
+Or the conversations around this topic in our [slack channel](https://cloud-native.slack.com/archives/C032LNSMY0P), even the [talk at Kubecon NA 2022](https://www.youtube.com/watch?v=Sdr5axlOnDI&list=PLj6h78yzYM2O5aNpRM71NQyx3WUe1xpTn&index=76) demonstrate the interest from the community in this feature.
+
+- What use cases does it support?
+
+Currently, buildpack authors can build and package their buildpacks for different OS and Architectures, but when they distribute them the URI for a buildpack can’t disambiguate, they need to use different tags to differentiate between them. This makes harder for users to consume those Buildpacks. 
+The solution is to share a single URI for all the different OS and Architectures, and the way to do that is using a manifest list.
+
+Adding commands to support the operations to handle the manifest list will allow buildpack authors to migrate their existing buildpacks to support multi-arch, without afecting their current existing process. 
+
+- What is the expected outcome?
+
+The expected outcome is to have a set of commands in pack to handle manifest lists, for example:
+- A command to create a manifest list from a set of images
+- A command to push the manifest list to a registry
+- A command to update or delete a manifest list
+
+# What it is
+[what-it-is]: #what-it-is
+
+The proposal is to add a new _experimental_ command `pack manifest` and different subcommands.   The `pack manifest` commands will initially be gated behind `pack config experimental`.  The `pack manifest` commands will move from experimental status to supported status when maintainers deem it appropriate.
+- `pack manifest create` will create a local manifest list for annotating and pushing to a registry
+- `pack manifest annotate` will add additional information like os, arch or variant to an existing local manifest list
+- `pack manifest add` will add an image to an existing manifest list
+- `pack manifest delete` will delete a manifest list from local storage
```

Comment:
We do not have a `delete` command. The `remove` command implements the functionality you have mentioned here. Podman have this same functionality in the `rm` command and it also have a `remove` command which is used to remove an image manifest from a local manifest list. So if we want to make the user experience same, we can rename our commands.